.eslintignore

```diff
@@ -1,12 +1,12 @@
-bin/**
```

Comment:
Could you revert all these changes? They are incorrect according to the gitignore specification and do not match [the behavior of node-ignore](https://github.com/kaelzhang/node-ignore/blob/master/test/ignore.js#L33-L56).